README.md

```diff
@@ -1,13 +1,99 @@
-# README
+# GitReps Community
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Installation and Usage Instructions
+
+## 1: Clone this repository
+
+```
+git clone https://github.com/GitRepsCommunity/platform.git
+```
+
+## 2: System Dependencies:
+To run this Rails project as intended you will need to install:
+- Ruby
+- Ruby on Rails
+- A Ruby version manager (optional but recommended)
+   - RVM
+   - chruby
+   - rbenv
+   - asdf
+
+## 3: HomeBrew installation:
+
+### Install Ruby:
+```
+brew install ruby
+```
+
+### Verifying Ruby version:
+Most Ruby projects will contain a .ruby-version file with the expected version of Ruby to run.
+You can change your version of Ruby using a version manager like RVM.
+
+### Installing a version manager for Ruby (RVM):
+As a first step install GPG keys used to verify installation package:
+```
+brew install gnupg 
+
+gpg --keyserver keyserver.ubuntu.com --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+```
+
+Install RVM (development version):
+```
+\curl -sSL https://get.rvm.io | bash
+```
 
-Things you may want to cover:
+View available versions of Ruby:
+```
+rvm list
+```
 
-* Ruby version
+Install specific version of Ruby (3.1.4 for this project as of 7/28/2023):
+```
+rvm install 3.1.4
+```
+#### NOTE: 
+If you are having installation/compilation issues with rvm install run:
+```
+rvm install 3.1.4 --with-openssl-dir={path to openssl@1.1}
+```
+
+You can find your openssl path on Linux/Unix systems by running:
+```
+which openssl
+```
+
+### Installing Rails:
+Recommended version: ```7.0.6```
```

Comment:
Similar to the Ruby version, we could just point people to the [Gemfile](https://github.com/GitRepsCommunity/platform/blob/1ff84fd34d71a99d242d062e91b151c1f57c0580/Gemfile#L7C24-L7C24) for the current version of Rails.